code/modules/projectiles/projectile_base.dm

```diff
@@ -1,3 +1,6 @@
+#define MOVES_HITSCAN -1 //Not actually hitscan but close as we get without actual hitscan.
+#define MUZZLE_EFFECT_PIXEL_INCREMENT 17 //How many pixels to move the muzzle flash up so your character doesn't look like they're shitting out lasers.
```

Comment:
Can you turn these into `///` comments and place them above the defines?

Reply:
done